Jenkinsfile

```diff
@@ -46,7 +46,7 @@ pipeline {
                 sh  '''
                     # Clone this repository & Parabank repository into the workspace
                     mkdir parabank-jenkins
-                    git clone https://github.com/whaaker/parabank-jenkins.git parabank-jenkins
+                    git clone -b Ramiro-SOA-WebUI https://github.com/whaaker/parabank-jenkins.git parabank-jenkins
```

Comment:
Your Ramiro-SOA-WebUI branch is going to go away after the merge, so you want this line to clone the main branch and not clone from your Ramiro branch while you were testing.